specs/phase1/beacon-chain.md

```diff
@@ -181,7 +185,17 @@ class ShardHeader(Container):
     # The actual data commitment
     commitment: DataCommitment
     # Proof that the degree < commitment.length
-    degree_proof: BLSCommitment
+    degree_proof: BLSKateProof
```

Comment:
Why the new type `BLSKateProof`? Note that a Kate proof is a Kate commitment to the quotient polynomial. So commitment is correct.

Also, in this particular case, the degree proof is no longer a Kate proof. So even if we want to use a new name for Kate proofs in the samples, in this case it should stay `BLSCommitment`.

Further, please use `KZG` instead of `Kate`.

Reply:
I'll revert this change for now, but hope to see more distinct types later. Documentation by typing is nice, it avoids a lot of mixups, and documents the purpose well.

> Further, please use KZG instead of Kate.

:+1: